index.html

```diff
@@ -4683,82 +4683,134 @@ <h2>Internationalization Considerations</h2>
     </p>
 
     <section class="informative">
-      <h3>Language Declarations</h3>
+      <h3>Language and Direction</h3>
 
       <p>
-When expressing human-readable text content in the data model, it is important
-for presentation, accessibility, and further processing to specify the language
-of the text, as follows:
+Data publishers are strongly encouraged to use the design pattern described
+in this section when expressing simple internationalized strings in
```

Comment:
> ...simple internationalized strings...

I'm not sure what these are. I'd probably say either just "strings" or I would clarify that we mean human language strings, which we normally call "natural language strings".

Reply:
Fixed in b2f3fdb952a9ca4a96411170bb5d465334cdcad3.